src/redux/reducers/configSlice.ts

```diff
@@ -53,13 +55,38 @@ const configSlice = createSlice({
               ? productDetermineMode
               : action.payload.mode || state.mode,
         }
+
+        // Remove _initialValues from the state temporarily, create it below
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        const { _initialValues, ...stateWithoutInitialValues } = loadedState
+
+        return {
+          ...stateWithoutInitialValues,
+
+          // _initialValues is a reference to the values that were used to load the widget initially.
+          // It is meant to be set, and then READ ONLY after that.
+          // Example:
+          // When a user dynamically changes the mode, use_cases, or data for a connection we need to
```

Comment:
Just curious, Would it be the user making these config changes, or the client? I'm trying to understand what kind of scenario might trigger a config reset.

Reply:
Upcoming, there is a whole new flow where the user will click on a prompt and if selected, that choice will change the mode, and the products, and the use cases.

Reply:
Ah, gotcha!

Reply:
Yeah, I was hoping this comment might help clarify.  It's hard to look into the future and know if you're not really close to the project, haha.  Thanks for bringing it up.